### ISSUE TITLE
env怎么配置呀

### ISSUE THREAD

**Issue by lqqla** [noodle-run/noodle#370]
![image](https://github.com/noodle-run/noodle/assets/98020030/a74dfc1d-1e61-41ab-bd5a-2cf5b8ac80b5)

**Comment by y4h2** [comment]
@lqqla 要分别在各个网站上注册:
- [NeonDB](https://neon.tech/)
- [Clerk Auth](https://clerk.dev/)
- [Upstash](https://upstash.com/)
- [Resend](https://resend.com/)

但是我感觉他这个项目根本没有内容, 我把所有的env都填上了, 代码也运行了, 但是没有任何功能.

**Comment by Erb3** [comment]
While I am unable to speak Chinese, I've translated your comments with Google Translate. Sorry for bad translations.

> But I feel like his project has no content at all. I filled in all the envs and the code ran, but there was no function.

Noodle is currently not in a working state, and is under development. This is also stated in the README.

> This is a work-in-progress and not the finished product. Noodle is still in active development towards a minimal viable product (MVP).

---

虽然我不会说中文，但我已使用 Google 翻译翻译了您的评论。抱歉翻译得不好。

> 但我觉得他的项目根本没有内容。我填写了所有环境，代码运行了，但没有任何功能。

Noodle 目前未处于工作状态，正在开发中。这也在 README 中说明。

> 这是一项正在进行的工作，而不是成品。Noodle 仍在积极开发中，旨在成为最小可行产品 (MVP)。

**Comment by y4h2** [comment]
@Erb3 Thanks for your reply. 
I can speak English, I just tried to answer @lqqla 's question, as his diagram uses Chinese.

Yes, I saw the README mentioned "This is a UI design mockup of what the platform will look like, it is not the current state of the project.". 
Sorry, I didn't mean to offend in the previous comment,  I just got a bit frustrated after registering 4 services and found besides the dashboard link redirection, nothing else works. As I saw the number of stars increase dramatically, I expected there might be some
killer features have been released.

I am looking forward to Noodle having its MVP sooner, really want to find some open-source projects that could beat Notion.

Best wishes.
